cloudwatchlogs_formatter.go

```diff
@@ -0,0 +1,172 @@
+package shuttle
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	v4 "github.com/aws/aws-sdk-go/aws/signer/v4"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+)
+
+const (
+	xAmazonTarget             = "Logs_20140328.PutLogEvents"
+	xAmazonJSON11             = "application/x-amz-json-1.1"
+	cloudWatchLogsServiceName = "logs"
```

Comment:
Took a look at the docs (https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutLogEvents.html) LGTM